libs/genai/tests/integration_tests/test_standard.py

```diff
@@ -84,6 +84,12 @@ def test_tool_message_histories_list_content(
     def test_usage_metadata_streaming(self, model: BaseChatModel) -> None:
         super().test_usage_metadata_streaming(model)
 
+    @pytest.mark.xfail(reason="investigate")
```

Comment:
I don't think just xfailing standard tests is a good idea.
Could you do an investigation, please?